com.github.emmanueltouzery.projectpad.json

```diff
@@ -0,0 +1,99 @@
+{
+  "app-id": "com.github.emmanueltouzery.projectpad",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "20.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "projectpad",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=dri",
+    "--filesystem=home",
+    "--share=network",
+    "--talk-name=org.freedesktop.secrets"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/projectpad/cargo"
+    }
+  },
+  "modules": [
+    {
+      "name": "gtksourceview-3.24.11",
+      "only-arches": [],
+      "buildsystem": "simple",
+      "build-commands": [
+        "./configure --prefix=/app",
+        "make install"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "sha256": "691b074a37b2a307f7f48edc5b8c7afa7301709be56378ccf9cc9735909077fd",
+          "url": "http://ftp.gnome.org/pub/gnome/sources/gtksourceview/3.24/gtksourceview-3.24.11.tar.xz"
```

Comment:
use download.gnome.org instead

Reply:
(FTR ftp.gnome.org redirects to download.gnome.org since a week or so.)

Reply:
Nice! it would be nice to avoid using it on the newly written manifests though